summary22.txt

```diff
@@ -0,0 +1,100 @@
+Bugfixes Fix failing unit tests for vehicle_turret
+PerformanceImprove performance of damage calculation
+Infrastructure Move crafting rig definitions to JSON
+Infrastructure Split engine load in to electrical and propulsion
+Infrastructure Update JSON serialization code
+Bugfixes Fix item groups' ammo/magazine chances not being applied
+Infrastructure Advance Inventory Filtering Brace Requirement Removal
+I18N Display localized ammo names in bandolier info
+I18N Display localized firing modes in UI
+I18N Extract GUNMOD locations as translatable string
+Infrastructure Overmap overhaul (step 4)
+Interface Use a wider type for symbols
+Balance Buff regen mutations, nerf regen due to health stat
+Build Allow installation in a path with whitespaces
+PerformanceDefer autopickup item lookups until first use.
+Bugfixes Fix autopickup UI's "S"witch option, which did nothing.
+Build Fix compilation issue for #19303
+Bugfixes Reveal some mission target locations on start
+Mods obsolete the version of PK_rebalancing included in the coregame
+Infrastructure Engine overspeed detection now uses vehicle speed in calculation
+Mods Convert Tanks and Other Vehicles mapgen overrides
+Bugfixes Fix NPCs not avoiding traps
+Content Add bone needle to "sewing_standard" requirement
+Infrastructure Extend "using" to recipes\other folder
+Infrastructure Remove wild entry in tool_armor.json
+Mods Updates to More Survival Tools
+Infrastructure Initialize bChar in the auto_pickup ctor.
+Balance Buffed 30x113 ammo belt capacity
+Bugfixes Prevent toggling of parts not currently enablable
+Bugfixes Fix unit tests for NPC needs
+PerformanceCache item types for inventory; huge crafting GUI speedup
+Bugfixes Fix craft requiring exact numbers of items
+Bugfixes Fix harvest drops not stacking
+Bugfixes Hack around long overflow in water crafting
+Build Add a compile mode that checks for printf format errors.
+Bugfixes Fix PARTIAL_DEAF Flag Support / Sound Processing Cleanup
+Balance Add recoil for being hit
+Build Add Linux to OSX cross-compilation. Take 2
+Bugfixes Don't give default names to corpses
+Bugfixes Enable "self-assigned" missions (no quest giver) and fix "goto" missions
+Bugfixes Fix corpse materials
+Bugfixes Re-Add Auto-Safe-Mode Coloring
+Balance Added copper to coilgun
+Content Sugar House mapgen
+Infrastructure Partially jsonize TALK_EVAC_GUARD3
+Bugfixes Fix fire vents
+Bugfixes Fix some wrongly generated plural strings
+Infrastructure Extend "using" to armor\storage recipes
+Infrastructure Extend "using" to armor\hands.json
+Bugfixes Fix a crash caused by map::add_splatter
+Bugfixes Fix for monster group pack size not working
+Infrastructure Modern Apartment JSONify
+Bugfixes Fix issue #19641 :Fix min volume for holsters using legacy rounding truncation (e.g. survivor utility belt)
+Bugfixes Down Blankets Allowed in Construction, resolves #19787
+Bugfixes Fix nones in items made by jmapgen_loot
+Infrastructure Add character radiation editing to debug menu [RDY]
```

Comment:
Would you please remove prefixes/suffixes like [RDY]? See https://github.com/CleverRaven/Cataclysm-DDA/pull/26008#issuecomment-426529752.